src/test/py/bazel/bazel_windows_test.py

```diff
@@ -554,6 +554,34 @@ def testTestShardStatusFile(self):
         ['test', '--incompatible_check_sharding_support', '//:foo_test']
     )
 
+  def testTestPrematureExitFile(self):
```

Comment:
Why add a separate test for Windows?

Reply:
`bazel_test_test` is disabled on Windows.

Reply:
Today I Learned! (or rather, re-learned :) )